packages/code-studio/src/styleguide/CheckboxGroups.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Checkbox, CheckboxGroup, Flex, Text } from '@deephaven/components';
```

Comment:
I think this is the incorrect `Checkbox` component. Need to use `SpectrumCheckbox`